contracts/libs/gov-pool/GovPoolStaking.sol

```diff
@@ -105,7 +96,7 @@ library GovPoolStaking {
 
             delegatorInfo.pendingRewards +=
                 (micropoolCumulativeSum - delegatorInfo.latestCumulativeSum).ratio(
-                    stakedAmount,
+                    currentDelegatorStake,
```

Comment:
I think that we might actually need to multiply by the `latestDelegatorsStake`. What do you think?

Reply:
done